fenix/app/src/androidTest/java/org/mozilla/fenix/experimentintegration/gradlewbuild.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import os
+import subprocess
+
+from .adbrun import ADBrun
+
+here = os.path.dirname(__file__)
+logging.getLogger(__name__).addHandler(logging.NullHandler())
+
+
+class GradlewBuild(object):
+    binary = "./gradlew"
+    logger = logging.getLogger()
+    adbrun = ADBrun()
+
+    def __init__(self, log):
+        self.log = log
+
+    def test(self, identifier):
+        # self.adbrun.launch()
```

Comment:
Nit: remove if not needed

Reply:
Oh whoops, this is only for testing locally